### PR TITLE
src/continue to clean up and unify file structure from PR50

### PR DESCRIPTION
**Pull request by patrycja-gomza**
Hi Team! 👋

This pull request addresses two changes:

1. Fix import path in MovieDetails:
Corrected the import path to movieSlice in the MovieDetails component.

2. Delete duplicated getGenreList file:
Removed the redundant file getGenreList as it was already present in the utils/API directory -- > according to @bedicooper #50 

Please review the changesand ensure that the modifications align with the project's coding standards. Thank you! 🙏

### CODE REVIEW

File: src/features/movies/MovieDetails/index.js
Comment (bedicooper) [inline src/features/movies/MovieDetails/index.js:1]: Thank You, I also noticed that bug now after merging.
I didn't catch that before and do not know why my liveserver had no isuess with ../ wile I worked on the changes.

File: src/features/movies/MovieList/getGenreList.js
Comment (bedicooper) [inline src/features/movies/MovieList/getGenreList.js:1]: this is very starange, when I move a file to a new folder and and than add that deletion (VSC deletes the old file and creates a new one) to staged changes for commit the VSC combines old and new files into one, but then sometimes after pushing commits to GH they do not disapear form their current location. Did You notice behavior like that in Your case?

**Review by bedicooper — APPROVED** [review]
Thank You for cleaning up and fixing that bug!

File: src/features/movies/MovieList/getGenreList.js
Comment (patrycja-gomza) [reply src/features/movies/MovieList/getGenreList.js:1]: Hi bedicooper 👋
I haven't encountered the exact behavior you described, but it seems like Visual Studio Code might be facing some issues when handling file movements and deletions. As far as I know, Git relies on file tracking, and sometimes changes related to file movements can be tricky for version control systems 😕

**Pull request merged**